test/parallel/test-https-options-boolean-check.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+const common = require('../common');
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const https = require('https');
+const fs = require('fs');
+
+const keyBuff = fs.readFileSync(`${common.fixturesDir}/keys/agent1-key.pem`);
```

Comment:
Ah OK, cool. Thanks, I'll change that over too. The original test file was copied over from another and amended, hence the previous inclusion of the license comments and usage of the `fs.readFileSync` method of including the pem files. Is there are issue/task that you're aware of to update the older tests to use the newer methods? Something like that I feel like would be a useful task for me to take up in future.